src/test/java/CliDisplayTest.java

```diff
@@ -0,0 +1,40 @@
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.*;
+
+import static org.junit.Assert.assertEquals;
+
+public class CliDisplayTest {
+
+    private final ByteArrayOutputStream output = new ByteArrayOutputStream();
+    private final ByteArrayInputStream input = new ByteArrayInputStream("string data\n".getBytes());
+    private PrintStream systemOutput;
+
+    @Before
+    public void beginIOStreams() {
+        systemOutput = System.out;
+        System.setOut(new PrintStream(output));
+        System.setIn(input);
+    }
+
+    @After
+    public void wipeStreams() {
+        System.setOut(systemOutput);
+        System.setIn(System.in);
+    }
```

Comment:
Your input method only belong to your beginIOStreams and systemOutput to wipeStreams.